cmd/vpu_plugin/vpu_plugin.go

```diff
@@ -156,16 +172,9 @@ func main() {
 	var sharedDevNum int
 
 	flag.IntVar(&sharedDevNum, "shared-dev-num", 1, "number of containers sharing the same VPU device")
-
 	flag.Parse()
 
 	klog.V(4).Info("debug is on")
```

Comment:
offtopic: now that I think of it, this is probably not very useful message

Reply:
remove it then. :-). done.